packages/block-editor/src/store/selectors.js

```diff
@@ -2082,7 +2083,7 @@ export const getInserterItems = createSelector(
 			{ core: [], noncore: [] }
 		);
 		const sortedBlockTypes = [ ...coreItems, ...nonCoreItems ];
-		return [ ...sortedBlockTypes, ...reusableBlockInserterItems ];
+		return [ ...sortedBlockTypes, ...syncedPatternInserterItems ];
```

Comment:
Does it still make sense to keep the "reusable blocks" in this selector, now that they're supposed to be treated as a pattern instead (and are part of the other selector)?

Reply:
Currently, the synced blocks still appear in the quick inserter list so are needed here. We can pull in from another selector of course,  but there are some inconsistencies with when/where the different patterns appear which need to be sorted out, [eg.](https://github.com/WordPress/gutenberg/issues/51945) so I suggest we leave this as is for this PR and circle back once we have finalised exactly how/when the different sync status types will be listed.